tests/foreman/api/test_ansible.py

```diff
@@ -0,0 +1,65 @@
+"""Test class for Ansible Roles and Variables pages
+
+:Requirement: Ansible
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Ansible
+
+:Assignee: sbible
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from wait_for import wait_for
+
+
+def test_fetch_and_sync_ansible_playbooks(target_sat):
+    """
+    Test Ansible Playbooks api for fetching and syncing playbooks
+
+    :id: 17b4e767-1494-4960-bc60-f31a0495c09f
+
+    :Steps:
+
+        1. Install ansible collection with playbooks.
+        2. Try to fetch the playbooks via api.
+        3. Sync the playbooks.
+        4. Assert the count of playbooks fetched and synced are equal.
+
+    :expectedresults:
+        1. Playbooks should be fetched and synced successfully.
+
+    :BZ: 2115686
+
```

Comment:
There's 1 customer scenario associated with the bz.  Please add `:customerscenario: true`